Cubical/ZCohomology/Groups/Sn.agda

```diff
@@ -391,3 +272,117 @@ Hⁿ-Sⁿ≅ℤ (suc n) =
   Ker-ϕ⊂Im-left theIso = K.Ker-d⊂Im-Δ  (suc n)
   Ker-right⊂Im-ϕ theIso = K.Ker-i⊂Im-d (suc n)
 
+
+
+
+{- More standard proof of H¹(S¹) ≅ ℤ using Mayer-Vietoris.
+This is much slower than the direct proof, but let's keep it here for completeness.
```

Comment:
I wonder if it would make sense to move this to Experiments? Or is there any reason why someone would use this proof instead of the direct one?

I definitely think we should keep this proof somewhere, especially if we plan to try to characterize these cohomology groups from the Eilenberg-Steenrod axioms. Most likely that proof will be similar to this one and I bet that a proof using those axioms instead of the concrete implementation of MV will be much faster.

Reply:
PS: I mean it would be good to have it uncommented in Experiments so that it doesn't break when we change the library

Reply:
his proof is a bit annoying since we don't have Guillaume's half-reduced version of Mayer-Vietoris (in which case it would be almost immediate) and should probably be improved at some point. But I guess it makes sense to keep it for future reference. I'll put it in experiments.